package com.arges.file.proto;    

message ServerHeart {
	optional string  		term 		= 1		[default = ""];  			//	server的当前周期数
	optional string  		identifier 	= 2		[default = ""];  			//	server的唯一标识符
	optional string  		content 	= 3		[default = ""];  			//	心跳具体信息
} 